app/views/director/counselors/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> Invite a counselor! </h1>
+
+<%= form_for @counselor, url: "/counselors" do |form| %>
```

Comment:
All good! Everything runs at it should :) The reason I did it this was was because this specific url doesn't have prefix !